package.json

```diff
@@ -19,7 +19,8 @@
     "posttest:onchain": "npm run network:down",
     "network:up": "node ./test/integration/support/after.js && node ./test/integration/support/before.js",
     "network:down": "node ./test/integration/support/after.js",
-    "lint": "eslint ."
+    "lint": "eslint .",
+    "generate-docs": "node_modules/.bin/jsdoc --configure jsdoc.json --verbose"
   },
   "husky": {
```

Comment:
You're using husky & lint-staged! Nice!

Reply:
to be very honest this is not me, it was here before. And looking at it, I'm not sure the hook works. But one thing at a time, I will inspect that after this PR.